app/content/carto-python/visualizing-data.md

```diff
@@ -0,0 +1,179 @@
+
+- **Color_continuous:** Helper function for quickly creating a color continuous style. Data values of each field are interpolated linearly across values in the domain and are then styled with a blend of the corresponding color in the range.
+
+
+For more information and details about function parameters, default values etc., please go to the [reference](https://docs.carto.com/carto-python/reference/).
+
+![png](/img/carto-python/carto-dw-notebook/cartodw_map.png)
+
+
+### Examples
+
+In this example, we visualize the spatial data with geometries in a map from our data warehouse connection established in our CARTO platform (could be to Google BigQuery, Snowflake, Redshift, Postgres, and Databricks).
+
+```python
+# Register CartoLayer in pydeck
+register_carto_layer()
+
+hexagons_query = """
+SELECT  CARTO_DEV_DATA.carto.H3_BOUNDARY("H3") H3_GEOM,
+        CROSSFIT_GYMS / TOTAL_GYMS AS dominance_ratio
+        FROM SFDATABASE.CARTO.GYMS_CA_TOTAL_CENTROID
+        """
+
+credentials = get_layer_credentials(carto_auth)
+
+hexagons = pdk.Layer(
+    "CartoLayer",
+    data = hexagons_query,
+    geo_column=pdk.types.String("H3_GEOM"),
+    type_=MapType.QUERY,
+    connection=pdk.types.String("my_carto_connection"),
+    credentials=credentials,
+    opacity=0.2,
+    stroked=True,
+    get_fill_color = color_continuous("DOMINANCE_RATIO", [x/10 for x in range(10)], colors = "Tropic"),
+    get_line_color=[0,42,42],
+    line_width_min_pixels=2,
+    on_data_error=get_error_notifier(),
+    )
+
+points_query = """
+SELECT GEOM, 'crossfit' AS CATEGORY
+FROM SFDATABASE.CARTO.GYMS_CA_CROSSFIT
+UNION ALL
+SELECT GEOM, 'competitors' AS CATEGORY
+FROM SFDATABASE.CARTO.GYMS_CA_COMPETITION
+"""
+
+points = pdk.Layer(
+    "CartoLayer",
+    data = points_query,
+    geo_column=pdk.types.String("GEOM"),
+    type_=MapType.QUERY,
+    connection=pdk.types.String("snowflake"),
+    credentials=credentials,
+    opacity=0.8,
+    stroked=True,
+    pickable=True,
+    point_radius_min_pixels=2,
+    get_fill_color = color_categories("CATEGORY", ["competitors", "crossfit"], colors = "Tropic"),
+    
+    on_data_error=get_error_notifier(),
+    )
+
+view_state = pdk.ViewState(latitude=33.64, longitude=-117.94, zoom=5)
+r = pdk.Deck(
+    [hexagons, points],
+    initial_view_state=view_state,
+    map_style=pdk.map_styles.LIGHT,
+)
+r.to_html(iframe_height = 700)
+```
+
+![png](/img/carto-python/styling-screenshot.png)
+
+
+</br>
+
+ In this case, you can see spatial data based on points visualized on a map, natively from our CARTO DW.  
+
+```python
+# Register CartoLayer in pydeck
+register_carto_layer()
+
+# Render CartoLayer in pydeck with color_bins style
+layer = pdk.Layer(
+    "CartoLayer",
+    data="SELECT geom, price_num FROM `carto-dw-ac-XXX.shared.01_listings_la_2021_5_reviews`",
```

Comment:
`carto-dw-ac-XXX` can be removed, because it uses this project as PROJECT_ID, so bigquery can find directly the shared dataset.